src/Campaigns/Entities/LandingPage/LandingPageFeatureItem.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Campaigns\Entities\LandingPage;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+class LandingPageFeatureItem extends AbstractEntity
+{
+    public const COLUMN_TITLE = 'title';
+    public const COLUMN_DESCRIPTION = 'description';
+    public const COLUMN_ICON = 'icon';
+    public const COLUMN_SIZE = 'size';
+    public const COLUMN_BUTTON_TEXT = 'buttonText';
+    public const COLUMN_BUTTON_URL = 'buttonUrl';
+
+    public const DEFAULT_VALUE_TITLE = '';
+    public const DEFAULT_VALUE_DESCRIPTION = '';
+    public const DEFAULT_VALUE_ICON = '';
+    public const DEFAULT_VALUE_SIZE = 4;
+    public const DEFAULT_VALUE_BUTTON_TEXT = '';
+    public const DEFAULT_VALUE_BUTTON_URL = '';
+
+    /**
+     * @var string
+     */
+    private $title;
```

Comment:
```suggestion
    private string $title;
```
And same for all new properties

Reply:
But why? That was not the case for the old ones? Why suddendly I have to type them like that? If I do that I have to do it on all other files ?!

Reply:
The repo now support  php 7.4 and above so I think it would be better to start using PHP7.4 syntax for new properties what do you think @pfeyssaguet?

Reply:
Yeah, let's make the front guy doing the PHP migration, good idea XD